examples/bzlmod/hello_world/MODULE.bazel

```diff
@@ -0,0 +1,18 @@
+module(
+    name = "hello_world",
+    toolchains_to_register = ["@rust_linux_x86_64__x86_64-unknown-linux-gnu//:toolchain"],
```

Comment:
Please use the `register_toolchain` directive instead of the `toolchains_to_register` attribute. Then you can group it together with the `use_rep` directive.

Reply:
See https://github.com/bazelbuild/bazel/issues/15829